Add information from http://wiki.ros.org/TestingRepository (#1905)
ros2/ros2_documentation: source/Installation/Prerelease-Testing.rst

Debian testing repository
-------------------------

> When packages are released into a rosdistro(using bloom), the buildfarm builds them into deb packages which are stored temporarily in the building apt repository.

Reviewer: ```suggestion
When packages are released into a ROS distribution (using bloom), the buildfarm builds them into debian packages which are stored temporarily in the **building** apt repository.
```

Reviewer: Also, maybe we should put a link to the bloom documentation here?

> As packages build during the day, an automatic process periodically synchronizes the packages in **building** to a secondary repository called **ros-testing**.

Reviewer: ```suggestion
As dependent packages are rebuilt, an automatic process periodically synchronizes the packages in **building** to a secondary repository called **ros-testing**.
```

**ros-testing** is intended as a soaking area where developers and bleeding-edge users may give the packages extra testing, before they are manually synced into the public ros repository from which users typically install packages.

> Approximately every two weeks, the rosdistro's release manager manually synchronizes the contents of **ros-testing** into **main** ros repository.

Reviewer: ```suggestion
Approximately every two weeks, the rosdistro's release manager manually synchronizes the contents of **ros-testing** into the **main** ROS repository.
```
